src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTree.php

```diff
@@ -44,7 +44,7 @@ class FileTree extends AbstractWidget
      *
      * @var string
      */
-    private $subTemplate = 'widget_filetree';
+    protected $subTemplate = 'widget_filetree';
```

Comment:
look at https://github.com/contao-community-alliance/dc-general/issues/333 ...
I have uploadet files from a frontend form and show the files at the backend - at a MM input mask... this works fine, but the editor should open the file (a PDF) via link in a new tab... and I override the template with my own